src/Filter/FilterField/MultipleValueAutocompleteFilterField.tsx

```diff
@@ -0,0 +1,60 @@
+import MenuItem from "@material-ui/core/MenuItem";
+import React from "react";
+
+import { SyntheticChangeEvent } from "../../../types/utils";
+import { MultipleValueAutocomplete } from "../../MultipleValueAutocomplete";
+import { useFilters } from "../context";
+import useStyles from "../styles";
+import { FilterData } from "../types";
+
+export interface FilterContentProps {
+  filter: FilterData;
+}
+
+export const MultipleValueAutocompleteFilterField: React.FC<FilterContentProps> =
+  ({ filter }) => {
+    const classes = useStyles();
+    const { onChange } = useFilters();
+
+    const { name, options } = filter;
+
+    if (options.choices === undefined) {
+      throw new Error("FilterType.Autocomplete must be used with choices prop");
```

Comment:
Instead of checking props at runtime we could instead change the `FilterDetailedOptions` type to [discriminated union](https://www.benmvp.com/blog/conditional-react-props-typescript/) or a [generic type](https://www.benmvp.com/blog/typescript-react-props-interfaces-type-aliases/) so that we stop invalid usage at build time